tests/tox.sh

```diff
@@ -33,6 +33,14 @@ if [ "${containers_to_remove}" ]; then
     docker rm -f $@ ${containers_to_remove} || echo failed to remove containers
 fi
 
+# copy the files to the root for the
+# types of images we're going to build
+mkdir -p {base,daemon,demo}
+# starting with kraken, the base image does not exist
+cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/base/* base || true
+cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/daemon/* daemon
+cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/demo/* demo
```

Comment:
sorry I forgot about this, chances are 'demo' might get included into daemon as well for kraken. Can you add `|| true` as well? Once you're done, feel free to merge. :)